deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -0,0 +1,45 @@
+{
+    "jobDefinitionName": "production-pfb-analysis-run-job",
```

Comment:
The production version bumps CPU/mem, as well as the postgresql settings in the environment section below, following the comments for each variable that are in setup_database.sh. I tended to err on the conservative side.